create-your-own-tutorial/index.asciidoc

```diff
@@ -1,4 +1,4 @@
-= Create a tutorial with the Tutorial-Compiler
+= Create your own tutorial with the Tutorial-Compiler
```

Comment:
Remove tutorial compiler. Nobody wanting to write a tutorial at first place needs to know about it.

Reply:
Done